test/yulPhaser/Chromosome.cpp

```diff
@@ -39,6 +49,24 @@ namespace solidity::phaser::test
 BOOST_AUTO_TEST_SUITE(Phaser)
 BOOST_AUTO_TEST_SUITE(ChromosomeTest)
 
+BOOST_AUTO_TEST_CASE(constructor_should_convert_from_string_to_optimisation_steps)
+{
+	vector<string> expectedSteps{
+		ConditionalSimplifier::name,
+		FunctionHoister::name,
+		RedundantAssignEliminator::name,
+		ForLoopConditionOutOfBody::name,
+		Rematerialiser::name,
+		ForLoopConditionOutOfBody::name,
+		ExpressionSimplifier::name,
+		ForLoopInitRewriter::name,
+		LoopInvariantCodeMotion::name,
+		ExpressionInliner::name
+	};
+
+	BOOST_TEST(Chromosome("ChrOmOsoMe").optimisationSteps() == expectedSteps);
```

Comment:
:)

Reply:
I wanted to do `soLiDiTy` but there's no `y` :(